benchexec/tools/svf.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for SVF: a framework for static value-flow analysis.
+    Specifically this tool is a wrapper around SVF to make it work with SV-COMP.
+    - Project URL: https://github.com/Lasagnenator/svf-svc-comp
```

Comment:
This is specified anyway below, it can be removed.

Reply:
Done.